x/stake/genesis.go

```diff
@@ -60,7 +60,7 @@ func InitGenesis(ctx sdk.Context, keeper Keeper, data types.GenesisState) (res [
 func WriteGenesis(ctx sdk.Context, keeper Keeper) types.GenesisState {
 	pool := keeper.GetPool(ctx)
 	params := keeper.GetParams(ctx)
-	validators := keeper.GetAllValidators(ctx)
+	validators := keeper.GetValidators(ctx, params.MaxValidators)
```

Comment:
This change would kill the export functionality. The point is to export all the validators are exported not just the top 100. Please re-add in the `GetAllValidators` function as it was before it was removed in this PR.